examples/examples_dotnet_test.go

```diff
@@ -59,6 +59,22 @@ func TestAzureContainerRegistryDotNet(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestAwsContainerRegistry(t *testing.T) {
+	region := os.Getenv("AWS_REGION")
+	if region == "" {
+		t.Skipf("Skipping test due to missing AWS_REGION environment variable")
+	}
```

Comment:
Is there a reason why we let this be environmentally controlled, instead of just hard coding it to a specific zone?

I can't see a scenario where we will want to change the zone our tests run in, but I might be missing something.

Reply:
Yeah, I see what you mean - the only thing I can think of is some sort of lazy failsafe that tries to capture both user error when running tests locally and CI failing to behave properly/regional AWS outage.

I'm not against just hard coding the region right here; but if that's a pattern we want to clean up we should probably do it across more than just this provider.